fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java

```diff
@@ -55,31 +54,42 @@ public void sendToUserAccount(String organisationName, String contactName, Strin
 
     @Override
     public void sendDefinedEmail(EmailDetail emailDetails) {
-        final Email email = new SimpleEmail();
         final SMTPCredentialsData smtpCredentialsData = this.externalServicesReadPlatformService.getSMTPCredentials();
 
         final String authuser = smtpCredentialsData.getUsername();
         final String authpwd = smtpCredentialsData.getPassword();
 
-        // Very Important, Don't use email.setAuthentication()
-        email.setAuthenticator(new DefaultAuthenticator(authuser, authpwd));
-        email.setDebug(false); // true if you want to debug
-        email.setHostName(smtpCredentialsData.getHost());
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost(smtpCredentialsData.getHost()); // smtp.gmail.com
```

Comment:
Rather than creating an instance of JavaMailSenderImpl directly here, wouldn't the right solution be to declare JavaMailSender as a bean and let Spring provide it through dependency injection?

Reply:
Yuh, we can follow that approach as well. Main intention was to resolve this email sending issue. I'll do that if this changes are approved by the community. Thanks for the idea :)

Reply:
Can you bring this to standard too as per @ptuomola suggestion?

Reply:
Is this necessary? As JavaMailSenderImpl is a class to send emails via the spring boot. (Not a customize one but you can have it after installing the necessary dependencies). I think this line is similar to this one `final Email email = new SimpleEmail();` (Needs to define this object as final)